### PR TITLE
Added two colors

### PR DESCRIPTION
**Pull request by dcparga**

### CODE REVIEW

File: Function.cs [hunk 1]
```diff
@@ -32,6 +32,7 @@ public class Function
             "Arsenic", 
             "As Dark As My Soul Black", 
             "Azure", 
+            "Baby blue",
```
Comment (javajudt) [inline Function.cs]: Someone else beat you to this one, it's down toward the bottom.
```suggestion

```

**Review by javajudt — CHANGES_REQUESTED** [review]
Baby blue is a duplicate. Remove that and we'll take your light olive green!

File: Function.cs (hunk 1 above)
Comment (dcparga) [reply Function.cs]: Done!

**Review by javajudt — APPROVED** [review]
Thanks!

**Pull request merged**